src/board_controller/emotibit/emotibit.cpp

```diff
@@ -7,14 +7,135 @@
 
 #include "custom_cast.h"
 #include "json.hpp"
-#include "network_interfaces.h"
 #include "timestamp.h"
 
+#ifdef __ANDROID__
```

Comment:
not related to pieeg, should be removed

Reply:
Reverted changes to emotibit.cpp - not related to pieeg.